addons/binding/org.openhab.binding.seneye/ESH-INF/thing/thing-types.xml

```diff
@@ -107,4 +111,28 @@
 		<description>The time your slide expires</description>
 		<state readOnly="true" />
 	</channel-type>
+	<channel-type id="wrongslide">
+		<item-type>String</item-type>
+		<label>Wrong slide</label>
+		<description>The Wrong Slide is in use (becomes 1 when the slide has expired)</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="slideserial">
+	   <item-type>String</item-type>
```

Comment:
Indentation contains spaces (also more below, simply Ctrl-Shift-F in eclipse in this file to re-format).